config/playground/helm/prod/helm-opensearch.yaml

```diff
@@ -30,6 +30,7 @@ opensearchHome: /usr/share/opensearch
 config:
   opensearch.yml: |
     cluster.name: opensearch-cluster
+    plugins.query.datasources.encryption.masterkey: ${QUERY_DATASOURCES_MASTER_KEY}
```

Comment:
can we remove Prod, this change will trigger Production deployment. we need to validate it in the dev environment first, and your observability

Reply:
Ok